conda.recipe/run_test.py

```diff
@@ -0,0 +1,14 @@
+import os
+import netCDF4
+
+# Check OPeNDAP functionality.
+url = 'http://geoport-dev.whoi.edu/thredds/dodsC/estofs/atlantic'
+nc = netCDF4.Dataset(url)
+
```

Comment:
The first two test above are *packaging tests* and may be removed. The test below is the actual unittests for `netCDF4-python`.